### PR TITLE
Add note about connecting to local Parker instance

### PR DESCRIPTION
**Pull request by mjangda**

### CODE REVIEW

**Review by rinatkhaziev — DISMISSED** [review]
Thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,20 @@ Tests are powered by [Jest](https://jestjs.io).
...
 ## Developing
 
+### Local Dev
+
+To test against a local instance of Parker, you can use the `API_HOST` env var. You'll also want to nullify the `VIP_PROXY` env var as well.
+
+Examples:
+
+```
+VIP_PROXY="" VIP_PROXY=localhost:4000 node ./dist/bin/vip
```
Comment (dchymko) [inline CONTRIBUTING.md]: Should this not be?:
```suggestion
VIP_PROXY="" API_HOST=localhost:4000 node ./dist/bin/vip
```

Comment (rinatkhaziev) [reply CONTRIBUTING.md]: oof, missed that 😅

Comment (mjangda) [reply CONTRIBUTING.md]: lol, good catch.

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**